ps_linklist.php

```diff
@@ -87,7 +88,7 @@ public function __construct()
         $this->description = $this->trans('Adds a block with several links.', array(), 'Modules.Linklist.Admin');
         $this->secure_key = Tools::encrypt($this->name);
 
-        $this->ps_versions_compliancy = array('min' => '1.7.5.0', 'max' => _PS_VERSION_);
+        $this->ps_versions_compliancy = array('min' => '1.7.7.0', 'max' => _PS_VERSION_);
```

Comment:
Sad to remove the compatibility with 1.7.5 & 1.7.6, more if we found a bug :(
If we keep the _legacy_link, can we keep the compliancy?

Reply:
Well the main purpose of this PR was to be able to test the new feature, but we are not obliged to merge since it doesn't bring new feature

I just thought it could be a nice example of how to use the feature and motivate people to upgrade to 177 at the same time ^^